types/node/package.json

```diff
@@ -181,6 +181,10 @@
         {
             "name": "Dmitry Semigradsky",
             "githubUsername": "Semigradsky"
+        },
+        {
+            "name": "Coder Shibe",
+            "githubUsername": "codershiba"
```

Comment:
Not sure that we're to the point of adding you as an owner.

Reply:
Too early to do so or some other reason?